lib/gossip/dissemination.js

```diff
@@ -182,7 +194,7 @@ Dissemination.prototype._issueAs = function _issueAs(filterChange, mapChanges) {
             continue;
         }
 
-        if (change.piggybackCount > this.maxPiggybackCount) {
+        if (change.piggybackCount >= this.maxPiggybackCount) {
```

Comment:
What happens if we delete the item before we have confirmation that it has been processed by the other side? I know that that cannot be done with `issueAsReceiver`, but `_issueAs` is also used by `issueAsSender`. With this change the changes might be removed before we have confirmation back.